protocol/driver.go

```diff
@@ -131,8 +140,37 @@ func (s *driverServer) Parse(rctx xcontext.Context, req *ParseRequest) (*ParseRe
 	return &resp, nil
 }
 
+func (s *driverServer) ServerVersion(rctx context.Context, _ *VersionRequest) (*Version, error) {
+	sp, ctx := opentracing.StartSpanFromContext(rctx, "bblfsh.server.Parse")
+	defer sp.Finish()
+
+	resp, err := s.d.Version(ctx)
```

Comment:
I'm not clear on why we have two distinct but structurally equivalent types here.

Reply:
The one is from the gRPC codegen, the second is defined by SDK. They happen to be the same so the direct cast is possible. If we change the type of time to a pointer, this won't be possible anymore so this code path will become clearer.